.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: labeler
```

Comment:
I think that it's good to leave a link to https://github.com/actions/labeler as a comment for future readers who don't know much about GitHub Actions.

Reply:
Thanks for the tip. Added!